src/requirements.txt

```diff
@@ -0,0 +1,8 @@
+pandas
```

Comment:
These should at least be pinned to versions for easier reproducibility, optionally could generate using [`--require-hashes`](https://pip.pypa.io/en/stable/topics/secure-installs/#hash-checking-mode).

Reply:
good TODO for you